<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="tableView">
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="action_Edit_Parameters"/>
    <addaction name="action_Edit_Headers"/>
   </widget>
   <widget class="QMenu" name="menuRows">
    <property name="title">
     <string>Rows</string>
    </property>
    <addaction name="actionInsert_Row_Above"/>
    <addaction name="actionInsert_Row_Below"/>
    <addaction name="actionDuplicate_Row"/>
    <addaction name="separator"/>
    <addaction name="actionRemove_Rows"/>
   </widget>
   <widget class="QMenu" name="menuColumns">
    <property name="title">
     <string>Columns</string>
    </property>
    <addaction name="actionAdd_Column_Right"/>
    <addaction name="actionAdd_Column_Left"/>
    <addaction name="separator"/>
    <addaction name="actionRemove_Columns"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuRows"/>
   <addaction name="menuColumns"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="action_Edit_Parameters">
   <property name="text">
    <string>Parameters</string>
   </property>
  </action>
  <action name="action_Edit_Headers">
   <property name="text">
    <string>Headers</string>
   </property>
  </action>
  <action name="actionInsert_Row_Above">
   <property name="text">
    <string>Add Above</string>
   </property>
  </action>
  <action name="actionInsert_Row_Below">
   <property name="text">
    <string>Add Below</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionRemove_Rows">
   <property name="text">
    <string>Remove Row(s)</string>
   </property>
  </action>
  <action name="actionAdd_Column_Right">
   <property name="text">
    <string>Add Right</string>
   </property>
  </action>
  <action name="actionAdd_Column_Left">
   <property name="text">
    <string>Add Left</string>
   </property>
  </action>
  <action name="actionRemove_Columns">
   <property name="text">
    <string>Remove Column(s)</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionDuplicate_Row">
   <property name="text">
    <string>Duplicate </string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
